Twig/Extension/StfalconTinymceExtension.php

```diff
@@ -153,6 +153,10 @@ public function tinymceInit($options = array())
                     // After parsing we add them together again.
                     $config['theme'][$themeName]['content_css'] = implode(',', $cssFiles);
                 }
+
+                if (isset($themeOptions['language_url'])) {
```

Comment:
@charlieWasp  You checking `$themeOptions['language_url']` but you `language_url` option specified before `arrayNode('theme')` so `isset($themeOptions['language_url'])` always return false.

You should move `->scalarNode('language_url')->defaultNull()->end()` in `->arrayNode('theme')` OR use `$config['language_url']` instead of `$themeOptions['language_url']`